### PR TITLE
use styled components for divs

### PR DESCRIPTION
**Pull request by ShaiahWren**
Continue making styled div components to take the place of inline styling. 

The UI should show no changes.

### CODE REVIEW

File: frontend/common/Masthead.tsx [hunk 1]
```diff
@@ -180,13 +188,13 @@ export function AnsibleMasthead(props: { hideLogin?: boolean }) {
                                     <AppBarDropdown icon={<CogIcon />}>
                                         <DropdownGroup label="Theme">
                                             <DropdownItem
-                                                icon={theme === ThemeE.Light ? <SunIcon /> : <div style={{ width: 24 }} />}
+                                                icon={theme === ThemeE.Light ? <SunIcon /> : <SunIconGroupDiv />}
```
Comment (ZitaNemeckova) [inline frontend/common/Masthead.tsx:191]: There's no `SunIconGroupDiv` or `MoonIconGroupDiv `. And you can use one definition for both `<div style={{ width: 24 }} />` replacements.

File: framework/PageMasthead.tsx [hunk 2]
```diff
@@ -11,6 +11,15 @@ import {
 import { BarsIcon } from '@patternfly/react-icons';
 import { Children, ReactNode } from 'react';
 import { usePageNavSideBar } from './PageNav/PageNavSidebar';
+import styled from 'styled-components';
+
+const ToolbarSpacingDiv = styled.div`
+  flex-grow: 1;
+`;
+
+const ToolbarDiv = styled.div`
```
Comment (ZitaNemeckova) [inline framework/PageMasthead.tsx:20]: You can do 
```js
const StyledToolbar = styled(Toolbar)`
  padding: 0px;
`;
```

File: frontend/common/Masthead.tsx (hunk 1 above)
Comment (ShaiahWren) [reply frontend/common/Masthead.tsx]: Now there is an `IconGroupDiv` for both components, and it is defined in a styled component that has been commented out.

**Review by ZitaNemeckova — APPROVED** [review]
LGTM :+1:

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**